app/src/main/kotlin/nyc/jsjrobotics/palmrgb/firebase/constants/FirebasePath.kt

```diff
@@ -0,0 +1,7 @@
+package nyc.jsjrobotics.palmrgb.firebase.constants
+
+object FirebasePath {
```

Comment:
Whats the point of this class with one variable?

Reply:
I'll message you on Slack